.github/workflows/e2e-parallel-full.yml

```diff
@@ -57,6 +57,19 @@ jobs:
           role-duration-seconds: 3600
           role-session-name: GithubActions-Session
 
+      - name: Iamlive Setup
+        run: |
+          #!/bin/bash
+          set -e
+          set -o pipefail
+          set -x
+          export IAMLIVE_VERSION=v0.48.0
```

Comment:
nit and personal opinion: what if we put this up top in the global env like https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/.github/workflows/pre-commit.yaml#L9